### PR TITLE
Layout2 publication and achievement page 

### PR DESCRIPTION
**Pull request by pmyip3**
# Description 
As title + change color combination 

# Type of change 
- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring
- [ ] Documentation 
- [x] Improvement 

# Problem
Current UI could be improved 

# Solution description
As below, also change some color combinations in both light and dark theme to make it consistent with the third-library Twitter feed built-in theme. 

Publications
- Use a list group
- Click to expand to view the description
- Link as 'PDF'

Achievements
- Use a list group

See more https://trello.com/c/gHuMXSET/152-layout-2-improvements

# Tests 
All unit tests pass? N/A

Builds successfully?Yes 

Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)
Light
image
image
image

image

Dark 
image
image
image
image
image

Provide steps in detail for reviewers to reproduce the expected result of the code changes.

# Relevant document/ link (if any) 
 https://trello.com/c/gHuMXSET/152-layout-2-improvements

# Risk
Low

### CODE REVIEW

File: scholly/base/src/layouts/layout2/components/landingPage/LandingPage.js [hunk 1]
```diff
@@ -20,13 +26,51 @@ const landingPage = () => {
...
-        <div className="landing-center-title">About Us</div>
-        <Image className="team-photo" src="https://images.pexels.com/photos/853168/pexels-photo-853168.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500" />
-        {homepageData.aboutUs.map((paragraph) => (
-          <div className="landing-center-content">{paragraph}</div>
-        ))}
-        <br />
+      <Container fluid className=" mt-3 mb-3">
+        <h2>
+          {' '}
+          Welcome to the
```
Comment (kdav108) [inline scholly/base/src/layouts/layout2/components/landingPage/LandingPage.js:32]: Nit: Can we remove `the`, to me 'Welcome to Monash Association of Coding' sounds a bit better than 'Welcome to the Monash Association of Coding'.

Comment (kdav108) [reply scholly/base/src/layouts/layout2/components/landingPage/LandingPage.js:32]: Or Alternatively how about titling this as `About {team name}` instead? I don't have a strong preference for either but just throwing it out there.

**Review by kdav108 — COMMENTED** [review]
Absolutely stunning 😍 design. Like the addition of recent publications to the About US page and the mobile friendliness.

I have a few UI suggestions (none of these are a must haves, we can implement the ones most people agree with):
1) Can we make the image on the Home bag in the centre or make the text start from next to the image or add the twitter feed next to the image. I say so because the top half of the Home page UI Is not very symmetrical atm.

2) Can we seperate each of the achievements with a greater gap, because both the publication list and team member list are in cards with gaps between them the achievement page looks relatively cluttered. Alternatively we can place achievement into cards similar to the publications list.

3) In Dark Mode the publication and achievement text is hard to read, it's fine when we hover on it but it makes it hard to skim through the list. Plus it becomes hard to realise the `Description` button is clickable.  I don't know what a good solution to this would be, but perhaps we can make the text permanently white like when we hover over it, and make the `description` div a seperate color? Maybe grey? Given this is not quite related to the current PR changes, if you want we can look into it later.

**Comment by pmyip3** [comment]
> Absolutely stunning 😍 design! I Like the addition of recent publications to the Home page and the mobile friendliness.
> 
> I have a few UI suggestions (none of these are a must haves, we can implement the ones most people agree with):
> **1)** Can we make the image on the Home page in the centre or make the text start from next to the image or add the twitter feed next to the image. I say so because the top half of the Home page UI Is not very symmetrical atm.
> 
> **2)** Can we seperate each of the achievements with a greater gap, because both the publication list and team member list are in cards with gaps between them the achievement page looks relatively cluttered. Alternatively we can place achievement into cards similar to the publications list.
> Screen Shot 2021-09-26 at 3 59 47 pm
> 
> **3)** In Dark Mode the publication and achievement text is hard to read, it's fine when we hover on it but it makes it hard to skim through the list. Plus it becomes hard to realise the `Description` button is clickable. I don't know what a good solution to this would be, but perhaps we can make the text permanently white like when we hover over it, and make the `description` div a seperate color? Maybe grey? Given this is not quite related to the current PR changes, if you want we can look into it later
> Screen Shot 2021-09-26 at 4 28 48 pm
> .


Thanks for the suggestion :)
1. I'll try to make the image in the center cause I think the other layout 2 is supposed to be making text under the image(correct me if I am wrong). Also, moving the Twitter feed up might not help the symmetrical issue as the Twitter handle isn't compulsory. 
2-3. Will work on that and upload the new UI in this PR.

**Pull request closed without merging**